tests/Doctrine/Tests/DBAL/Platforms/MySQL57PlatformTest.php

```diff
@@ -14,6 +14,22 @@ public function createPlatform()
         return new MySQL57Platform();
     }
 
+    public function testHasNativeJsonType()
+    {
+        $this->assertTrue($this->_platform->hasNativeJsonType());
+    }
+
+    public function testReturnsJsonTypeDeclarationSQL()
+    {
+        $this->assertSame('JSON', $this->_platform->getJsonTypeDeclarationSQL(array()));
```

Comment:
@phansys is that really important here? We would need to fix all of that stuff in one PR anyways.